db/migrate/20240207131735_create_appts.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAppts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :appts do |t|
```

Comment:
Use proper name. Don't use short forms fot table names

Reply:
since the name of project is appointment cant use the table name appointment